src/main/java/de/propra/timetracker/TimeTrackerCLI.java

```diff
@@ -46,6 +46,10 @@ CLIStatus readCLI(String[] args) throws IOException {
         options.addOption(tableOption);
         Option tableOptionDate = new Option(null, "table-of-date", true, "Zeige Tabelle aller Einträge eines bestimmten Datums");
         options.addOption(tableOptionDate);
+        Option trackStart = new Option(null, "start", true, "Starte automatisches Tracking");
+        options.addOption(trackStart);
+        Option trackStop = new Option(null, "stop", true, "Beende automatisches Tracking");
+        options.addOption(trackStop);
```

Comment:
Hier bitte genauer reinschreiben wie die Verwendung ist. In der README ist das gut gelöst, aber wenn man nun `--help` auf der Konsole aufruft, ergibt es nicht so viel Sinn und man weiß nicht was man da noch schreiben muss. 

Das sollten wir perspektivisch auch noch bei den anderen Punkten ergänzen. Die Hilfe soll ja erklären wie ich nun die Parameter da eingebe etc. :-) Die README ist ja eher was für Devs.